File: sandbox/ruby/src/ApiAppDeleteDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
end

begin
    Dropbox::Sign::ApiAppApi.new.api_app_delete(
        "0dd3b823a682527788c4e40cb7b6f7e9",
    )
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling ApiApp#api_app_delete: #{e}"
end

File: sandbox/node/src/ApiAppDeleteDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.ApiAppApi();

const clientId = "0dd3b823a682527788c4e40cb7b6f7e9";

apiCaller.apiAppDelete(
    clientId,
).catch(error => {
  console.log("Exception when calling ApiApp#apiAppDelete:");
  console.log(error.body);
});
